windows/setup.ps1

```diff
@@ -42,15 +42,18 @@ Check-Call { C:\Python37\python -m pip install -r requirements.txt  }
 Check-Call { C:\ProgramData\chocolatey\choco install git -y }
 Check-Call { C:\ProgramData\chocolatey\choco install 7zip -y }
 Check-Call { C:\ProgramData\chocolatey\choco install cmake -y }
-Check-Call { setx PATH "$($env:path);c:\Program Files\CMake\bin" }
 Check-Call { C:\ProgramData\chocolatey\choco install ninja -y }
 
 # Deps
 Check-Call { C:\Python37\python  windows_deps_headless_installer.py }
 
 # Other software
-#Check-Call { C:\ProgramData\chocolatey\choco install jom -y }
-#Check-Call { C:\ProgramData\chocolatey\choco install mingw -y }
+Check-Call { C:\ProgramData\chocolatey\choco install jom -y }
```

Comment:
do we need jom? I changed to ninja.

Reply:
No, if you changed the builds to use ninja, we no longer need it. I'll delete it.